packages/context/src/interceptor.ts

```diff
@@ -104,6 +104,33 @@ export class InvocationContext extends Context {
     });
   }
 
+  /**
+   * The target class, such as `OrderController`
+   */
+  get targetClass() {
+    return typeof this.target === 'function'
+      ? this.target
+      : this.target.constructor;
+  }
+
+  /**
+   * The target name, such as `OrderController.prototype.cancelOrder`
+   */
+  get targetName() {
+    return DecoratorFactory.getTargetName(this.target, this.methodName);
+  }
+
+  /**
+   * Description of the invocation
+   */
+  get description() {
+    return `InvocationContext(${this.name}): ${this.targetName}`;
+  }
+
+  toString() {
```

Comment:
Isn't `InvocationContext(${this.name}): ${this.targetName}` already a string? What would be the difference between description and `toString()`?

Reply:
`description` can be used in debugging, such as `invocationCtx.description`. `toString` overrides the base method so that it uses `description` as the string representation of the invocation context object.